HIRS_Utils/src/main/java/hirs/tpm/TPMBaselineGenerator.java

```diff
@@ -591,16 +591,15 @@ private void parseBaseline(final String baselineName, final InputStream inStream
             throws IOException, TPMBaselineGeneratorException {
         BufferedReader reader = new BufferedReader(new InputStreamReader(
                 inStream, "UTF8"));
-        String dataRow = reader.readLine();
+        String dataRow = "";
```

Comment:
Kind of a nitpick, but you don't need to initialize this with the empty String since that value gets overwritten on the first iteration of the `while` loop below. Could just be `String dataRow;`

Reply:
I agree that it seems guaranteed that dataRow will be overwritten in the while loop, but `String dataRow;` throws an error in the first catch block at `if (dataRow != null)` that dataRow may not have been initialized.

Reply:
Ah, good call. In that case, why not do `reader.readLine()` here, then for the while loop instead of `while(true)` do `while((dataRow != null && dataRow.matches("[0-9a-zA-Z./()_,\" -]+")`, and then move the `dataRow = reader.readLine()` that's in the `while` loop to the end of the loop instead of the beginning. Avoids this whole debacle with a small restructuring.

Reply:
Debacle avoided.